lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -97,7 +97,7 @@ def calculate_option_line_lengths
             width ||= text
                         .split("\n")
                         .reject(&:empty?)
-                        .map { |l| (l.length / max_width).ceil }
+                        .map { |l| (CLI::UI.fmt(l, enable_color: false).length / max_width).ceil }
```

Comment:
Why the `enable_color: false`?

Reply:
oh! because that's the bug

Reply:
Actually, not quite.  It'll miss-count due to ansi codes instead without that.